src/preamble.js

```diff
@@ -558,11 +558,40 @@ function instrumentWasmTableWithAbort() {
 }
 #endif
 
+#if LOAD_SOURCE_MAP
+var wasmSourceMap;
+#include "source_map_support.js"
```

Comment:
Looks like this is still included from `runtime_shared.js` - can we not end up including it twice?